hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -246,4 +246,9 @@ private OMConfigKeys() {
       "ozone.om.enable.filesystem.paths";
   public static final boolean OZONE_OM_ENABLE_FILESYSTEM_PATHS_DEFAULT =
       false;
+
+  public static final long OZONE_OM_MAX_TIME_TO_WAIT_FLUSH_TXNS =
+      TimeUnit.MINUTES.toSeconds(5);
+  public static final long OZONE_OM_FLUSH_TXNS_RETRY_INTERVAL_SECONDS =
+      TimeUnit.SECONDS.toMillis(5);
```

Comment:
For OZONE_OM_FLUSH_TXNS_RETRY_INTERVAL_SECONDS , here should be 5. On RatisUpgradeUtils#waitForAllTxnsApplied, we already do the convert:
>long intervalTime = TimeUnit.SECONDS.toMillis(timeBetweenRetryInSeconds);

Reply:
Thank you, fixed!